Misc/NEWS.d/next/Library/2019-10-16-08-08-14.bpo-38490.QbDXEF.rst

```diff
@@ -0,0 +1 @@
+Covariance, Pearson's correlation, and simple linear regression functionality was added to statistics module
```

Comment:
Please add a period at the end of the sentence.

Also, add "Patch by Tymoteusz Wołodźko" at the end! :)